addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -40,6 +40,7 @@ export class KanbanRenderer extends Component {
         this.state = useState({
             columnQuickCreateIsFolded:
                 !this.props.list.isGrouped || this.props.list.groups.length > 0,
+            isResequenceInprogress: false,
```

Comment:
As it doesn't need to be reactive, putting it in the state is unnecessary, right?
`this.isResequenceInprogress = false` should do it :slightly_smiling_face:

Reply:
Done! Thanks :slightly_smiling_face:

Reply:
Actually we need this to be responsive when `isResequenceInprogress` is true then no-content helper should be remove from view vice versa :slightly_smiling_face: